Products/PleiadesEntity/skins/PleiadesEntity/place_view.pt

```diff
@@ -118,6 +118,15 @@
         <p tal:condition="not:portal_state/anonymous">Red titles indicate drafting-state items. Yellow titles indicate pending-state.</p>
 
         <div tal:condition="not:context/@@iterate/baseline|nothing">
+
+          <div id="representativePointField" class="field">
+            <label>Representative Point:</label>
+            <dl tal:replace="structure context/@@representative-point"></dl>
```

Comment:
This is a super-nitpicky and dumb stylistic concern, but I would make this a div instead of a dl. It's replaced in the end, so it doesn't matter at all, but the others use dl because the rendered output is an actual list of some sort, but in this case it's not a list but a single data point.